phylogenetic/rules/merge_inputs.smk

```diff
@@ -84,8 +91,12 @@ def _gather_inputs():
         raise InvalidConfigError("Names of inputs (config.inputs and config.additional_inputs) must be unique")
     if not all(['name' in i and ('sequences' in i or 'metadata' in i) for i in all_inputs]):
         raise InvalidConfigError("Each input (config.inputs and config.additional_inputs) must have a 'name' and 'metadata' and/or 'sequences'")
+    available_keys = set(['name', 'metadata', 'sequences'])
+    if any([len(set(el.keys())-available_keys)>0 for el in all_inputs]):
+        raise InvalidConfigError(f"Each input (config.inputs and config.additional_inputs) can only include keys of {', '.join(available_keys)}")
 
-    return {i['name']: _parse_config_input(i) for i in all_inputs}
+    all_input_paths = itertools.chain(*[[v for k,v in el.items() if k!='name'] for el in all_inputs])
```

Comment:
this line (and the import) can be dropped, they're left over from a previous implementation